src/data/boundary-landing.json

```diff
@@ -1,167 +1,85 @@
 {
-  "heading": "Boundary Overview",
-  "subheading": "Use Boundary to create granular, identity-based access controls to manage user access across dynamic infrastructures.",
-  "blocks": [
-    {
-      "type": "heading",
-      "level": 2,
-      "size": 400,
-      "heading": "Explore Boundary Documentation"
+  "hero": {
+    "heading": "Boundary Placeholder Duplicate Of Vault Content",
+    "image": "https://place-hold.it/600x320"
+  },
+  "overview": {
+    "heading": "What is Boundary?",
+    "body": "Secure, store and tightly control access to tokens, passwords, certificates, encryption keys for protecting secrets and other sensitive data using a UI, CLI, or HTTP API.",
+    "cta": {
+      "text": "Learn more about Boundary features",
+      "url": "/vault/docs/what-is-vault"
     },
-    {
-      "type": "get_started",
-      "product": "boundary",
-      "heading": "Introduction to Boundary",
-      "text": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Iaculis dui sed venenatis, purus non augue arcu. Faucibus viverra praesent velit id vitae placerat gravida. Quis in nullam.",
-      "link": {
-        "text": "Learn More",
-        "url": "/boundary/docs/what-is-boundary"
+    "image": "https://place-hold.it/680x250"
+  },
+  "get_started": {
+    "heading": "Getting Started",
+    "body": "Use Boundary to securely store, access, and manage secrets and other sensitive data.",
+    "ctas": [
+      {
+        "text": "CLI Quick Start",
+        "url": "/vault/tutorials/getting-started"
```

Comment:
It would be great if we could source this data from the API. But I understand if for speed we don't do that now.

Reply:
Perhaps we could re-use the `getInlineCollections` functions from the product tutorial views and just do a basic slug array here?

Reply:
That's definitely an option! Will certainly be doing this for `TutorialCards` (and `CollectionCards` if that comes up; it's not in designs but I assume it will probably come up soon).

I'm slightly more hesitant to do it here for `cta` links... I think it might be important for authors to have the option to link to non-Tutorial content in the `get_started` block (not sure though!). And it seems they might want to be able to link to either collections or tutorials (flat slug array would be potentially ambiguous, I think). Since this content is specific to `dev-portal`, I figured that using `dev-portal` URL structures shouldn't be too brittle (will only need to be updated if we refactor our URL structure again).

All that being said, could see the case for a future enhancement anyways - eg a `cta` object could be either `{ text, url }` for non-tutorial links, or `{ tutorialSlug: "" }` for tutorial links, or `{ collectionSlug: ""}` for collection links. And then in `getStaticProps` those latter two formats get baked into `{ text, url }` with tutorial or collection data respectively. Let me know if this latter approach sounds like what you had in mind (Then I'll make a ticket to explore using tutorial & collection slugs for `cta` links more generally - not just for `cards`!)